### PR TITLE
Correction names and set level logger

### PR DESCRIPTION
**Pull request by Amilcar-B**

### CODE REVIEW

File: COMPILER_SERVICE/src/model/execute.py [hunk 1]
```diff
@@ -1,6 +1,4 @@
 import subprocess
```
Comment (FreddyClaros) [inline COMPILER_SERVICE/src/model/execute.py:1]: I think there's missing the header, added please

**Review by FreddyClaros — APPROVED** [review]
Looks good to me, Good job

File: CONVERT_SERVICES/src/configLogConvert/logconfigconvert.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 import os
```
Comment (aquena-code) [inline CONVERT_SERVICES/src/configLogConvert/logconfigconvert.py:18]: The Copyright name has other name

File: DataBase/src/configLogDataBase/log_config_Data.py [hunk 3]
```diff
@@ -17,7 +17,7 @@
 import os
```
Comment (aquena-code) [inline DataBase/src/configLogDataBase/log_config_Data.py:18]: The Copyright name has other name

File: MACHINE_LEARNING_SERVICE/src/configLogMachineLearning/logconfigmachine.py [hunk 4]
```diff
@@ -17,7 +17,7 @@
 import os
```
Comment (aquena-code) [inline MACHINE_LEARNING_SERVICE/src/configLogMachineLearning/logconfigmachine.py:18]: The Copyright name has other name

File: REPORTING_SERVICE/src/configLogReporting/logconfigreporting.py [hunk 5]
```diff
@@ -17,7 +17,7 @@
 import os
```
Comment (aquena-code) [inline REPORTING_SERVICE/src/configLogReporting/logconfigreporting.py:18]: The Copyright name has other name

File: rest_api/resources/configlogRestApi/logconfigrestapi.py [hunk 6]
```diff
@@ -17,7 +17,7 @@
 import os
```
Comment (aquena-code) [inline rest_api/resources/configlogRestApi/logconfigrestapi.py:18]: The Copyright name has other name

File: BOOKING_SERVICE/src/configLogBooking/logconfigbooking.py [hunk 7]
```diff
@@ -17,7 +17,7 @@
 import os
```
Comment (aquena-code) [inline BOOKING_SERVICE/src/configLogBooking/logconfigbooking.py:18]: The Copyright name has other name and miss a empty line

**Review by aquena-code — CHANGES_REQUESTED** [review]

**Pull request merged**